auto_tune_vllm/execution/backends.py

```
@@ -283,13 +292,65 @@ def poll_trials(self, job_handles: List[JobHandle]) -> Tuple[List[TrialResult],
+
+        logger.info(f"Cleaning up {len(self.active_actors)} active trials...")
+
+        # Call cleanup_resources on all active actors with timeout
+        cleanup_futures = []
+        for job_id, actor in self.active_actors.items():
+            try:
+                # Call cleanup asynchronously with timeout
+                cleanup_ref = actor.cleanup_resources.remote()
+                cleanup_futures.append((job_id, cleanup_ref))
+                logger.info(f"Initiated cleanup for trial {job_id}")
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)